sdk/textanalytics/ai-text-analytics/src/transforms.ts

```diff
@@ -317,3 +317,188 @@ export function transformError(errorResponse: unknown): any {
     statusCode: strongErrorResponse.statusCode,
   });
 }
+
+function toHealthcareResult(
+  documents: TextDocumentInput[],
+  results: GeneratedHealthcareResult
+): HealthcareResult[] {
+  function makeHealthcareEntity(entity: GeneratedHealthcareEntity): HealthcareEntity {
+    const { dataSources, ...rest } = entity;
+    return {
+      dataSources: dataSources ?? [],
+      ...rest,
+    };
+  }
+  function makeHealthcareRelation(
+    entities: HealthcareEntity[]
+  ): (relation: HealthcareRelation) => HealthcareEntityRelation {
+    return (relation: HealthcareRelation): HealthcareEntityRelation => ({
+      relationType: relation.relationType,
+      roles: relation.entities.map(
+        (role: HealthcareRelationEntity): HealthcareEntityRelationRole => ({
+          entity: entities[parseHealthcareEntityIndex(role.ref)],
+          name: role.role,
+        })
+      ),
+    });
+  }
+  return transformDocumentResults<HealthcareEntitiesDocumentResult, HealthcareSuccessResult>(
+    documents,
+    results,
+    {
+      processSuccess: ({ entities, relations, ...rest }) => {
+        const newEntities = entities.map(makeHealthcareEntity);
+        return {
+          entities: newEntities,
+          entityRelations: relations.map(makeHealthcareRelation(newEntities)),
+          ...rest,
+        };
+      },
+    }
+  );
+}
+
+function toCustomEntityRecognitionResult(
```

Comment:
maybe I misunderstand, but this function is called toCustomEntityRecognitionResult but looks like it's wrapping the customSingleLabelClassificationResult

Reply:
Good point, I renamed it accordingly.